r/R/dplyr.R

```diff
@@ -631,17 +671,85 @@ abandon_ship <- function(call, .data, msg = NULL) {
   eval.parent(call, 2)
 }
 
-arrange.arrow_dplyr_query <- function(.data, ...) {
+arrange.arrow_dplyr_query <- function(.data, ..., .by_group = FALSE) {
+  call <- match.call()
+  exprs <- quos(...)
+  if (.by_group) {
```

Comment:
This is subtle: so "by_group" sorting means adding group vars to the sort. Can you leave a comment? (I was confused about this at first.)

Reply:
Yep. Comment added in 1eb791afb23ffc770dd37d93d7b5b4b2a0484945.